lib/common/operations.c

```diff
@@ -44,83 +48,157 @@ pcmk__op_key(const char *rsc_id, const char *op_type, guint interval_ms)
     return crm_strdup_printf(PCMK__OP_FMT, rsc_id, op_type, interval_ms);
 }
 
-gboolean
-parse_op_key(const char *key, char **rsc_id, char **op_type, guint *interval_ms)
+static gboolean
+try_basic_match(const char *key, char **rsc_id, char **op_type, guint *interval_ms)
 {
-    char *notify = NULL;
-    char *mutable_key = NULL;
-    char *mutable_key_ptr = NULL;
     size_t len = 0, offset = 0;
-    unsigned long long ch = 0;
-    guint local_interval_ms = 0;
-
-    // Initialize output variables in case of early return
-    if (rsc_id) {
-        *rsc_id = NULL;
-    }
-    if (op_type) {
-        *op_type = NULL;
-    }
-    if (interval_ms) {
-        *interval_ms = 0;
-    }
+    size_t op_len = 0;
 
-    CRM_CHECK(key && *key, return FALSE);
-
-    // Parse interval at end of string
     len = strlen(key);
     offset = len - 1;
+
+    // Parse interval at end of string
     while ((offset > 0) && isdigit(key[offset])) {
-        ch = key[offset] - '0';
-        for (int digits = len - offset; digits > 1; --digits) {
-            ch = ch * 10;
-        }
-        local_interval_ms += ch;
         offset--;
     }
+
     if (interval_ms) {
-        *interval_ms = local_interval_ms;
+        unsigned long long l;
```

Comment:
I think unsigned long would be more consistent with strtoul() and the guint cast (and ditto later below)

Reply:
Done.